### PR TITLE
Add some notes in the blocker FAQ about unsupported repos

### PR DESCRIPTION
**Pull request by troglodyne**
case RE-158: Hopefully this reduces the amount of issue filings on that.

By submitting pull requests to this repo, I agree to the Contributor License Agreement which can be found at: https://github.com/cpanel/elevate/blob/main/docs/cPanel-CLA.pdf

### CODE REVIEW

File: docs-website-src/content/blockers.md [hunk 1]
```diff
@@ -149,3 +149,15 @@ These issues with the YUM repositories can cause ELevate to block your upgrade:
   * If YUM is in an unstable state (running `yum makecache` fails).
 
 If any unfinished yum transactions are detected, ELevate will attempt to complete them by executing `/usr/sbin/yum-complete-transaction --cleanup-only`. If this fails, ELevate will block you from beginning the upgrade process until you manually resolve any outstanding issues or transactions.
+
+#### Unsupported Repository
+
+By default, our stance on package repositories is that if they are not in a specific list of "vetted" repos,
+we will not allow upgrade while packages from said repository are installed.
+This is because of the following reasons:
+  * We have no way of knowing whether the repositories even have equivalent packages for the version the distribution we are upgrading to.
```
Comment (timmullin) [inline docs-website-src/content/blockers.md:158]: Maybe:
"the version **of** the distribution"
instead of:
"the version the distribution"

Comment (troglodyne) [reply docs-website-src/content/blockers.md:158]: Filed DOCS-261 for phrase review internally

**Comment by troglodyne** [comment]
Docs updated per case in question, force pushed. Should be good to go now

**Comment by cPholloway** [comment]
The CI refuses to run with a forked workflow.  I've asked Andy to resubmit this from the main project and I'm going to go ahead and close this request in favor of the one from the main PR.

FWIW: This change looks good so it is just a matter of semantics at this point

**Pull request closed without merging**